h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -791,6 +791,18 @@ public RegressionModelPrediction predictRegression(RowData data, double offset)
     return p;
   }
 
+  public KLimeModelPrediction predictKLime(RowData data) throws PredictException {
```

Comment:
this should also be added to the predict() method in this class

Reply:
done: https://github.com/h2oai/h2o-3/pull/4618/commits/e1f353f5837fa2c12a2049d0b063d8b87d58f7a4